designs/virtualization/4.5/Scheduling/Affinity/Affinity.md

```diff
@@ -0,0 +1,63 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Affinity
+
+Introducing Affinity as it appears in-page and the Edit modal.
+
+## in-page placement
+
+![Affinity field in the Scheduling section ](img/Affinity-0-0.jpg)
+The affinity field is placed in the Workload details page, Details tab, Scheduling section, Policies columns.
+
+![Affinity field in the Scheduling section ](img/Affinity-0-1.jpg)
+The affinity field counts the number of rules applied.
+
+## Affinity rules list
+
+![Affinity all rules modal ](img/Affinity-1-0.jpg)
+On clicking on the counter Or the Edit pencil icon, the user will be presented with a modal, containing all the applied rules for this workload.
```

Comment:
```suggestion
On clicking on the counter or the Edit pencil icon, the user will be presented with a modal, containing all the applied rules for this workload.
```

Reply:
maybe I'm missing something, why are we adding both a pencil and the counter as a way to get to the modal? Wasn't this changed to just the pencil?

Reply:
I don't recall dropping the clickability of the counter. But I will remove it so it won't create confusion.